packages/components/src/internal/components/samples/SamplesTabbedGridPanel.spec.tsx

```diff
@@ -201,15 +201,15 @@ describe('SamplesTabbedGridPanel', () => {
     });
 
     test('canPrintLabels false', () => {
-        const wrapper = mountWithAppServerContext(<SamplesTabbedGridPanel {...DEFAULT_PROPS} canPrintLabels={false} />);
+        const wrapper = mountWithAppServerContext(<SamplesTabbedGridPanel {...DEFAULT_PROPS} />, {}, { user: TEST_USER_READER}, {}, {}, {canPrintLabels: false});
         validate(wrapper);
         expect(wrapper.find(TabbedGridPanel).prop('supportedExportTypes')).toBeUndefined();
         expect(wrapper.find(TabbedGridPanel).prop('onExport')).toBeUndefined();
         wrapper.unmount();
     });
 
     test('canPrintLabels true', () => {
```

Comment:
update names of tests to be `showLabelOption false` and `showLabelOption true`?

Reply:
add a couple additional tests to cover both